src/Processors/AugmentProperties.php

```diff
@@ -70,6 +70,11 @@ public function __invoke(Analysis $analysis): void
     protected function augmentSchemaType(Analysis $analysis, OA\Schema $schema): void
     {
         $context = $schema->_context;
+
+        if (!$context->reflector || $context->is('nested')) {
```

Comment:
`!$context->reflector` should be `null === $context->reflector` instead (avoiding type juggling)